### PR TITLE
core-node-docs: HTTP response

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Description

This PR adds documentation on the HTTP response node, as there is currently no documentation on this node in the core-node-docs.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by ZJvandeWeg** [comment]
@sumitshinde-84 Can you fill out the form in the PR? Description, related issues, etc?

File: src/_includes/core-nodes/http-response-use-case.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## What is the Http Response node in Node-RED?
+
+The HTTP Response node allows you to send tailored HTTP responses back to clients that make requests to your application. It enables you to customize status codes, headers, and response bodies, facilitating dynamic communication between your application and its users.
+
+## Configuring HTTP Response node
+
+- **Status code:** This field allows you to specify the HTTP status code to be included in the response, you can also set that with the `msg.statusCode`
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/http-response-use-case.md:7]: Link to: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status?

File: src/_includes/core-nodes/http-response-use-case.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- **Headers:** with this feild you can add any custom HTTP headers to be included in the response. Headers provide additional information about the data being transferred between the client and server. You might use headers to specify the content type (Content-Type), set cookies, or provide caching directives. to set headers dynamically you can use `msg.headers` with JSON object containing headers.
+- **Cookies:** To set cookies, you need to pass `msg.cookies` to the response node. The `msg.cookies` object should contain key-value pairs where the keys represent the names of the cookies, The value can be either a string to set the value of the cookie with default options, or it can be an object of options
+
+The valid options include:
+
+- **domain:** (String) Optional. Specifies the domain name for the cookie.
+- **expires:** (Date) Optional. Specifies the expiry date for the cookie in GMT. If not specified or set to 0, it creates a session cookie.
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/http-response-use-case.md:14]: A session cookie can expire too, maybe you want to use that as example and write the implication of picking 0 explicitly e.g. no expiry

**Comment by sumitshinde-84** [comment]
> @sumitshinde-84 Can you fill out the form in the PR? Description, related issues, etc?

I have added the description, but regarding issues, there are currently none related to this PR.

File: src/_includes/core-nodes/http-response-use-case.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+## What is the Http Response node in Node-RED?
```
Comment (ZJvandeWeg) [inline src/_includes/core-nodes/http-response-use-case.md:1]: HTTP is an abbreviation that's always in caps
```suggestion
## What is the HTTP Response node in Node-RED?
```

**Pull request merged**